README.md

```diff
@@ -1,2 +1,29 @@
-# labels
-Copies labels into a new repo
+# 🏷 Labels
+Copies labels from one repo to another
```

Comment:
Sync lables between GitHub repositories.
Because sometimes we _want_ everything to be **_exactly_ the same**.
![image](https://cloud.githubusercontent.com/assets/194400/19173271/6e55c5e8-8c1e-11e6-9c4b-46a88d924e75.png)